doczrc.js

```diff
@@ -47,30 +47,29 @@ export default {
   },
   menu: [
     "Introduction",
+    "Roadmap",
     {
-      name: "Identity",
+      name: "DSNP",
       menu: [
         "Overview",
+        "Identity",
+        "Identifiers",
+        "Graph",
+        "Batch Publications",
+        "Announcements"
       ]
     },
-    "Identifiers",
-    "Graph",
     {
-      name: "Batch Publications",
-      menu: [
-        "Overview",
-      ],
-    },
-    {
-      name: "Announcements",
+      name: "Activity Content",
       menu: [
         "Overview",
       ]
     },
+    "DSNP Implementations",
     {
-      name: "Activity Content",
+      name: "DSNP: Ethereum",
```

Comment:
This should really be nested under `DSNP Implementations`, but docz does not support sub-nested menus #141. For now, I think this is fine until that issue gets resolved.